lab-katherine/README.md

```diff
@@ -0,0 +1,9 @@
+![cf](https://i.imgur.com/7v5ASc8.png) Lab 15: Queue
```

Comment:
Add a little bit more information about returns values and error cases. 

For example: What happens if I do enqueue(undefined);?